tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -1,19 +1,112 @@
 jobs:
 - job: Windows_CI_Dev
-  pool: Win-CPU
```

Comment:
why is the agent pool gone?

Reply:
If it's removed,  the same build definition can be used in different vsts accounts, without modification.  Also, you can deploy a VSTS agent on your local host, and use that for debugging.